schema.sql

```diff
@@ -1,5 +1,17 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+/* Create database */
+CREATE DATABASE vet_clinic ;
+
+-- connect database 
+\c vet_clinic
+
+/* create table */
+ CREATE TABLE animals(
+    id                 INT GENERATED BY DEFAULT AS IDENTITY,
+    name               VARCHAR(100) NOT NULL,
+    date_of_birth      DATE NOT NULL,
+    escape_attempts    INT,
+    neutered           BIT,
+    weight_kg          DECIMAL) ;
+
```

Comment:
- [ ] Please make sure to add a column `species` of type string to the `animals` table as requested in the project requirements.
   - You can consider using:  `ALTER TABLE animals ADD species varchar(255);`

Reply:
Hi, @thecodechaser can you kindly use this command line `SELECT * FROM animals;` to verify? It's already there.